tsctl

```diff
@@ -130,4 +131,5 @@ if __name__ == '__main__':
     shell_manager.add_option(
         u'-c', u'--config', dest=u'config', default=u'/etc/timesketch.conf',
         required=False)
+    shell_manager.add_command(u'runserver', Server(host="127.0.0.1", port=5000))
```

Comment:
* Use single quotes and unicode for strings, "127.0.0.1" -> u'127.0.0.1'
* Move this line above the "add_option" to group together with the rest of the commands